manager/controllers/app/m4dapplication_controller_unit_test.go

```diff
@@ -55,8 +54,10 @@ func TestM4DApplicationControllerCSVCopyAndRead(t *testing.T) {
 	}
 
 	// Register operator types with the runtime scheme.
-	s := scheme.Scheme
-	err = app.AddToScheme(scheme.Scheme)
+	s := runtime.NewScheme()
+	err = corev1.AddToScheme(s)
+	g.Expect(err).NotTo(gomega.HaveOccurred())
+	err = app.AddToScheme(s)
 	g.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
We don't really test AddToScheme errors. Can we have a single function that creates a scheme, similar to:
```
s := runtime.NewScheme()
_ = corev1.AddToScheme(s)
...
_ = app.AddToScheme(s)
return s

```

Reply:
Sure. We can extract a method that creates a custom scheme for us and registers the most common apis.